static/templates/settings/display_settings.hbs

```diff
@@ -59,6 +59,14 @@
                             <img class="emoji" src="/static/generated/emoji/images-{{this.key}}-64/1f389.png" />
                             {{/if}}
                         </span>
+                        {{#if (eq this.key "google-blob")}}
+                        <span>(<a href='/help/emoji-and-emoticons#change-your-emoji-set'>deprecated</a>)</span>
+                        <i
+                          class="fa fa-question-circle-o tippy-zulip-tooltip"
+                          data-tippy-placement="bottom"
```

Comment:
Let's move the placement into the `tippyjs.js` file -- if you're going to have custom styling for its tooltip anyway, best to have it all in one place.